Makefile

```diff
@@ -241,6 +243,82 @@ catalog-deploy:
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/2-loki.yaml	 | kubectl --ignore-not-found=true  delete -f -  || true
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml | kubectl --ignore-not-found=true  delete -f -  || true
+	-pkill --oldest --full "3100:3100"
+
+.PHONY: deploy-grafana
+deploy-grafana: ## Deploy grafana.
+	@echo "### Deploying grafana"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+	./hack/deploy-grafana.sh $(NAMESPACE)
+	-pkill --oldest --full "3000:3000"
+	kubectl port-forward --address 0.0.0.0 svc/grafana 3000:3000 2>&1 >/dev/null &
+	@echo -e "\ngrafana ui is available (user: admin password: admin) on http://localhost:3000\n"
+
+.PHONY: undeploy-grafana
+undeploy-grafana: ## Undeploy grafana.
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+	kubectl delete --ignore-not-found=true route grafana
+	kubectl delete --ignore-not-found=true service grafana
+	kubectl delete --ignore-not-found=true deployment grafana
+	kubectl delete --ignore-not-found=true configMap grafana-datasources
+	-pkill --oldest --full "3000:3000"
+
+.PHONY: ocp-deploy
+ocp-deploy: manifests generate fmt lint deploy-loki deploy-grafana install deploy-sample-cr   ## OCP deploy (loki, grafana and example-cr)
+	oc expose service grafana || true
+	@grafana_url=$$(oc get route grafana -o jsonpath='{.spec.host}'); \
+	echo -e "\nAccess grafana on OCP using: http://"$$grafana_url"\n"
+	oc expose service loki || true
+	@loki_url=$$(oc get route loki -o jsonpath='{.spec.host}'); \
+	echo -e "\nAccess loki on OCP using: http://"$$loki_url"\n"
+
+.PHONY: ocp-undeploy
+ocp-cleanup: undeploy-loki undeploy-grafana uninstall undeploy-sample-cr   ## OCP cleanup
+	-PID=$$(pgrep --oldest --full "main.go"); pkill -P $$PID; pkill $$PID
+
+.PHONY: ocp-run
+ocp-run: ocp-cleanup ocp-deploy   ## OCP-deploy + run the operator locally
+	@echo "### Running operator locally (in background process)"
+	-PID=$$(pgrep --oldest --full "main.go"); pkill -P $$PID; pkill $$PID
+	go run ./main.go &
+	@echo "### Waiting for goflow-kube pod to be ready"
+	while : ; do kubectl get deployment goflow-kube && break; sleep 1; done
+	kubectl wait --timeout=120s --for=condition=ready pod -l app=goflow-kube
+	@echo "### Enable network-observability-plugin in OCP console"
+	oc patch console.operator.openshift.io cluster --type='json' -p '[{"op": "add", "path": "/spec/plugins", "value": ["network-observability-plugin"]}]'
+	@echo "### Starting log-flows export into goflow-kube pod"
+	GF_IP=`oc get svc goflow-kube -o jsonpath='{.spec.clusterIP}'` && \
+	echo "goflow-kube pod ID: $$GF_IP" && \
+	oc patch networks.operator.openshift.io cluster --type='json' -p "[{'op': 'add', 'path': '/spec', 'value': {'exportNetworkFlows': {'ipfix': { 'collectors': ['$$GF_IP:2055']}}}}]"
```

Comment:
You don't need to patch network.operator any more since CNO now watches ovs-flows-config configmap.

Reply:
@memodi if I understood correctly ... this will be enabled only on OCP 4.10, so for anyone that uses an older version (e.g. 4.9 == current), this will not work. We can sure remove this once 4.10 GA's

Make sense ??? the idea is to allow quick and easy development experience